web/accounts/oauth.go

```diff
@@ -28,10 +34,6 @@ func (a *apiAccount) Links() *jsonapi.LinksList {
 }
 
 func start(c echo.Context) error {
-	if !middlewares.IsLoggedIn(c) {
```

Comment:
Will it still work in the browser without session code ?

Reply:
Yes, it will still work without a session code if the user is logged in